extensions/api/auth-spi/src/main/java/org/eclipse/dataspaceconnector/api/auth/AuthenticationRequestFilter.java

```diff
@@ -32,10 +34,11 @@ public AuthenticationRequestFilter(AuthenticationService authenticationService)
     public void filter(ContainerRequestContext requestContext) {
         var headers = requestContext.getHeaders();
 
-        var isAuthenticated = authenticationService.isAuthenticated(headers.entrySet().stream().collect(Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)));
-
-        if (!isAuthenticated) {
-            throw new NotAuthorizedException();
+        if (!OPTIONS.equalsIgnoreCase(requestContext.getMethod())) {
```

Comment:
nit: a small comment indicating credentials are excluded in preflight requests would be helpful